### ISSUE TITLE
NameError: global name 'requests' is not defined

### ISSUE THREAD

**Issue by zesaver** [xarnze/sublimepushbullet#1]
Hello,

I am trying to use the Sublime Pushbullet Plugin with ST3. It was installed successfully, I have added my key at the first launch. But I can not push anything. The console says following:

```
Exception in thread Thread-76:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "Pushbullet in /home/zesaver/.config/sublime-text-3/Installed Packages/Pushbullet.sublime-package", line 199, in run
NameError: global name 'requests' is not defined

Traceback (most recent call last):
  File "Pushbullet in /home/zesaver/.config/sublime-text-3/Installed Packages/Pushbullet.sublime-package", line 60, in <lambda>
  File "Pushbullet in /home/zesaver/.config/sublime-text-3/Installed Packages/Pushbullet.sublime-package", line 66, in handle_threads
AttributeError: 'ApiCall' object has no attribute 'result'
```

Could you give me any advice about making the plugin working? Thank you!

**Comment by xarnze** [comment]
It looks like the dependencies haven't installed correctly, could you try removing and the re-installing with the console open and posting the install log?

Thanks

**Comment by xarnze** [comment]
Fixed in 338b0d5 there was a missing import for requests if you update from package control it should be working now. Let me know how you get on

**Comment by zesaver** [comment]
Hello,

I restarted ST3 and in a few seconds there was a print in the console about Pushbullet update. So now everything works perfectly and I do can push from Sublime Text to my devices.

Thank you!
